### ISSUE TITLE
Using Reagent for guestbook tutorial

### ISSUE THREAD

**Issue by p-adams** [luminus-framework/examples#1]
I would be nice to see examples of using Reagent to render the views in the guestbook tutorial instead of the templates.

**Comment by yogthos** [comment]
Agreed, I keep meaning to add that, but just never get around to it. I do have an example of that in my book, and the code is available [here](https://pragprog.com/titles/dswdcloj2/source_code).
